sophiread/CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ add_subdirectory(FastSophiread)
 # add_subdirectory(SophireadCLI)
 
 # Add Sophiread stream command line interface
-# add_subdirectory(SophireadStreamCLI)
+add_subdirectory(SophireadStreamCLI)
```

Comment:
It is possible to switch the library used in the streaming CLI app to `FastSophiread` (the benchmark apps in `FastSophiread` should be a good reference), and you will no longer need to link to `SophireadLib`.

Reply:
Ok, that future PR I mentioned above would add a replacement for SophireadStreamCLI.